recipes/tcsh/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu -o pipefail
+
+./configure --prefix="${PREFIX}"
+make
```

Comment:
For `readline`, we added `SHLIB_LIBS="$(pkg-config --libs ncurses)"` after `make`. This was needed as we break out `libtinfo` from `libncurses`.

ref: https://github.com/conda-forge/readline-feedstock/blob/7ab893396747c916cf2e1dd71713baa5018471b4/recipe/build.sh#L4